### PR TITLE
Document productions productions

### PR DESCRIPTION
**Pull request by jdrieghe**
### Added
- Documentation for `/productions` related endpoints

### Changed
- Added `PRODUCTIES_AANMAKEN` permission to `/permissions` documentation

---
Issue: https://jira.uitdatabank.be/browse/III-3360

### CODE REVIEW

**Comment by bertramakers** [comment]
Thanks @jdrieghe, looks good! 👍 
I've gone ahead and also added a `Productions` tag to the relevant endpoints so they are grouped together in Swagger-UI. The description of the tag is super generic but consistent with the others. We'll improve all of that later when we tackle the Swagger docs improvements in general.

**Comment by jdrieghe** [comment]
@konrad126 Can you add the documentation for the endpoints you added to this PR?

File: swagger.json [hunk 1]
```diff
@@ -4441,7 +4441,30 @@
...
+        "/productions/suggestion/": {
+            "get": {
+                "summary": "Get next suggestion pair",
+                "description": "Authentication required",
+                "produces": [
+                    "application/ld+json"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "A list of permissions",
```
Comment (bertramakers) [inline swagger.json:4454]:
```suggestion
                        "description": "A pair of suggested similar events",
```

File: swagger.json [hunk 2]
```diff
@@ -6447,17 +6470,35 @@
...
+                        {
                         "type": "string",
                         "description": "A uuid identifying an event",
                         "example": "13518167-4582-4724-820a-d4ef0bff7976"
-                    }
+                        },
+                        {
+                        "type": "string",
+                        "description": "A uuid identifying an event",
+                        "example": "13518167-4582-4724-820a-d4ef0bff7976"
```
Comment (bertramakers) [inline swagger.json:6512]: Why would we duplicate this item?

File: swagger.json [hunk 3]
```diff
@@ -6447,17 +6470,35 @@
...
                 "eventIds"
             ]
+        },
+        "SuggestionPair": { 
+            "properties": {
+                "events": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                         "$ref": "#/definitions/Permission"
```
Comment (bertramakers) [inline swagger.json:6528]:
```suggestion
                         "$ref": "#/definitions/Event"
```

File: swagger.json [hunk 4]
```diff
@@ -6499,6 +6529,24 @@
...
+                "eventIds": {
+                    "type": "array",
+                    "minItems": 2,
+                    "uniqueItems": true,
+                    "items": 
+                        {
+                        "type": "string",
+                        "description": "A uuid identifying an event",
+                        "example": "13518167-4582-4724-820a-d4ef0bff7976"
+                        }
```
Comment (bertramakers) [inline swagger.json:6544]:
```suggestion
                    }
```

File: swagger.json [hunk 5]
```diff
@@ -6499,6 +6529,24 @@
...
             }
+        },
+        "SkipSuggestionPair": {
+            "properties": {
+                "eventIds": {
+                    "type": "array",
+                    "minItems": 2,
+                    "uniqueItems": true,
+                    "items": 
+                        {
```
Comment (bertramakers) [inline swagger.json:6540]: This should be on the line above, i.e. `"items": {`

File: swagger.json [hunk 6]
```diff
@@ -6524,8 +6524,7 @@
                 "events": {
                     "type": "array",
                     "items": {
-                        "type": "string",
-                         "$ref": "#/definitions/Permission"
+                         "$ref": "#/definitions/Event"
```
Comment (bertramakers) [inline swagger.json:6522]:
```suggestion
                        "$ref": "#/definitions/Event"
```

**Pull request merged**